avro-converter/src/test/avro/EnumUnion.avsc

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "record",
```

Comment:
Again, should we wrap a `UNION` within a `STRUCT` or just have a UNION data type?